python/src/otel/otel_sdk/otel_wrapper.py

```diff
@@ -17,32 +18,53 @@
 # resource = Resource.create().merge(AwsLambdaResourceDetector().detect())
 # trace.get_tracer_provider.resource = resource
 
-
-def _load_distros():
+def _load_distros() -> BaseDistro:
     for entry_point in iter_entry_points("opentelemetry_distro"):
         try:
-            entry_point.load()().configure()  # type: ignore
-            logger.info("Distribution %s configured", entry_point.name)
+            distro = entry_point.load()()
+            if not isinstance(distro, BaseDistro):
+                logger.debug(
```

Comment:
Should we create a follow up issue to find out why these debug logs can’t be seen in Lambda? But print statements CAN be seen in Lambda?

Reply:
yes please. No idea why the log disappear now.